content/docs/1.6.0/deploy/important-notes/index.md

```diff
@@ -106,3 +106,26 @@ Starting from `v1.5.0`, the following deprecated custom resource fields will be
 - BackingImageManager.status.backingImageFileMap.directory
 - BackingImageManager.status.backingImageFileMap.downloadProgress
 - BackingImageManager.status.backingImageFileMap.url
+
+### Engine Upgrade Enforcement
+
+Starting from `v1.6.0`, Longhorn will implement mandatory version upgrades for the engines. The minimum required supporting version will be outlined in the [release note](https://github.com/longhorn/longhorn/releases/tag/v{{< current-version >}}).
+
+When performing a helm upgrade, incompatible versions are automatically detected and prevented through a pre-hook mechanism. However, if you've installed Longhorn using manifests, the enforcement is handled by the longhorn-manager. This will result in unsuccessful pod launches and corresponding error logs. If you encounter such an error, it's necessary to revert to the previous version and upgrade any engine currently utilizing the incompatible engine image.
+
+You can assess the engine image versions for your current engines by utilizing the script below:
+```bash
+#!/bin/bash
+
+namespace="longhorn-system"
+
+engine_images=$(kubectl -n $namespace get engineimage -o=jsonpath='{.items[*].metadata.name}')
+
+for engine_image in $engine_images; do
+    cli_api_version=$(kubectl -n $namespace get engineimage $engine_image -o=jsonpath='{.status.cliAPIVersion}')
+    controller_api_version=$(kubectl -n $namespace get engineimage $engine_image -o=jsonpath='{.status.controllerAPIVersion}')
+    echo "EngineImage: $engine_image | cliAPIVersion: $cli_api_version | controllerAPIVersion: $controller_api_version"
+done
+```
+
+After successfully upgrading to version 1.6.0, the engine image versions will be accessible in the UI for subsequent version upgrades.
```

Comment:
Suggested wording: 
Once you successfully upgrade to version `v1.6.0`, you will be able to view information about engine image versions on the UI.